.changeset/good-cats-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': patch
```

Comment:
As discussed with @mayank99, although the iTwinUI-css API has changed with the removal of `.iui-dialog-animation-enter` and `.iui-dialog-animation-enter-active`, this is made a patch change instead of a minor. 

The reason for this is that these two classes were only meant to be consumed on the React side. The CSS used to work and continues to work correctly even without these classes.